### ISSUE TITLE
Unexpected token error

### ISSUE THREAD

**Issue by Immortalin** [beatrichartz/csv#21]
I am getting this error:

when parsing this:

```csv
 Message
"prof·li·gate\ˈprä-fli-gət, -ˌgāt\
adjective
: carelessly and foolishly wasting money, materials, etc. : very wasteful
Full Definition
1 : wildly extravagant <profligate spending>
2 : completely given up to dissipation and licentiousness <leading a profligate life>
prof·li·gate·ly adverb
Origin: Latin profligatus, from past participle of profligare to strike down, from pro- forward, down + -fligare (akin to fligere to strike); akin to Greek phlibein to squeeze.
First use: 1617
Synonyms: extravagant, high-rolling, prodigal, spendthrift, squandering, thriftless, unthrifty, wasteful
Antonyms: conserving, economical, economizing, frugal, penny-pinching, scrimping, skimping, thrifty
Synonyms: fritterer, high roller, prodigal, spender, spendthrift, squanderer, waster, wastrel
Antonyms: economizer, penny-pincher
2
prof·li·gate\ˈprä-fli-gət, -ˌgāt\
noun
: a person given to wildly extravagant and usually grossly self-indulgent expenditure
Origin: (see 1profligate ).
First use: 1709
Synonyms: extravagant, high-rolling, prodigal, spendthrift, squandering, thriftless, unthrifty, wasteful
Antonyms: conserving, economical, economizing, frugal, penny-pinching, scrimping, skimping, thrifty
Synonyms: fritterer, high roller, prodigal, spender, spendthrift, squanderer, waster, wastrel
Antonyms: economizer, penny-pincher"
"46984364136: Beamish
adjective
```
and I am getting this error:

```Elixir
%SyntaxError{description: "unexpected token: \":\" (column 61, codepoint U+003A)",
 file: "nofile", line: 4}
```

and this is my code:

```Elixir

defmodule SpellingListParser do
  File.stream!("SpellingList.csv") |> CSV.decode :headers: true |> Enum.map fn row ->
    Enum.each(row, IO.puts)
  end
end
```

**Comment by beatrichartz** [comment]
I'll have a look at this next week, just having a superficial look at it now it looks like you're running into the same issue as #13

**Comment by Immortalin** [comment]
@beatrichartz thanks!

**Comment by beatrichartz** [comment]
I tried to reproduce this issue, however the resulting error is not the same for me with the input and code given - I receive `(CSV.Parser.SyntaxError) Unterminated escape sequence. on line 1` since this line:

```
"prof·li·gate\ˈprä-fli-gət, -ˌgāt\
```
has no terminating double quote - line breaks do not get parsed as per RFC as described in #13 .

If you can update the information I have in https://github.com/beatrichartz/csv/tree/issue-21 / provide a pr against it, I can have another look at what exactly is causing the issue.

**Comment by beatrichartz** [comment]
Released multiline support with `1.3` - please have a look! Feel free to reopen if the issue persists.
